### DISCUSSION TITLE
WhatsApp mockup screen with messages and chats using daisyUI and sveltekit

### DISCUSSION THREAD

**Discussion by anhsirk0** [saadeghi/daisyui#1158] [Show and tell]
Hi! I just wanted to share a project .

![new2](https://user-images.githubusercontent.com/59538102/191080173-266da252-5870-4407-b352-6f9968bb4b20.gif)

This is a WhatsApp Mockup window that you can use to create fake welcome/notification messages.

[Source code](https://github.com/anhsirk0/whatsapp-svelte)

**Comment by Mashiane** [comment]
Wow awesome. Your repo also have something Ive been needing in a long time but incomplete.

If you dont mind I will request help from the DaisyUI team. Hopefully we can get this going.

**Reply by anhsirk0** [reply]
What is incomplete? Can you tell me so that I can try to improve.

**Reply by Mashiane** [reply]
No its not this project, please see this: https://github.com/saadeghi/daisyui/discussions/1321

**Reply by anhsirk0** [reply]
ok
